### DISCUSSION TITLE
vasp2wan90_vs2_fix for vasp5.4.4 interface with Wannier90 v.2.x or v.3.x?

### DISCUSSION THREAD

**Discussion by quynhxinhdep** [Chengcheng-Xiao/VASP2WAN90_v2_fix#23] [General]
- I read from Vaspwiki  "Up to VASP 6.1.x you need to set -DVASP2WANNIER90 to interface with Wannier90 v.1.x, and -DVASP2WANNIER90v2 for Wannier90 v.2.x, and add the Wannier90 library to [makefile.include]
Since VASP 6.2.0 you need to set -DVASP2WANNIER90 to interface with Wannier90 v.2.x or v.3.x."
 - so I wonder vasp2wan90_vs2_fix for vasp5.4.4 interfaces with Wannier90 v.2.x or v.3.x?

**Comment by Chengcheng-Xiao** [comment]
This patch works for both Wannier90 v.2.x or v.3.x, but only for VASP v5.4.4.
